test/common.sh

```diff
@@ -97,3 +108,25 @@ function test_setup() {
   echo ">> Uploading test images..."
   ${REPO_ROOT_DIR}/test/upload-test-images.sh || return 1
 }
+
+# Retrieve latest version from given Knative repository tags
+# On 'main' branch the latest released version is returned
+# On 'release-x.y' branch the latest patch version for 'x.y.*' is returned
+# Similar to hack/library.sh get_latest_knative_yaml_source()
+function get_latest_release_version() {
+    local repo_name="$1"
+    local major_minor=""
+    if is_release_branch; then
+      local branch_name
+      branch_name="$(current_branch)"
+      major_minor="${branch_name##release-}"
+    fi
+    local version
+    version="$(git ls-remote --tags --ref https://github.com/knative/"${repo_name}".git \
+      | grep "${major_minor}" \
+      | cut -d '-' -f2 \
+      | cut -d 'v' -f2 \
+      | sort -Vr \
```

Comment:
Are you sure that `-V` works on every platform's sort ? (cool find btw, didn't know the option before)

Reply:
It's used in release.sh script on a few occasions, that's my source. :)
I'll check the compatibility. However, the script isn't meant to be used outside of our CI.

Reply:
The `sort` seemed to work fine even without `-V` flag for the `semver` scheme.

Reply:
And `-V` it isn't posix compatible of course, it's a bit newer. :)

Reply:
> The sort seemed to work fine even without -V flag for the semver scheme.

Nope, I don't think so for larger patcher version numbers:

```
echo -n "1.2.3\n1.2.10\n1.2.5" | sort
1.2.10
1.2.3
1.2.5
```

but I'm totally fine with using `-V` if it is already used (and working) elsewhere.